src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration.Tests/ConfigurationLiveTests.cs

```diff
@@ -73,8 +80,7 @@ private async Task<string> SetMultipleKeys(ConfigurationClient service, int expe
         [Test]
         public async Task DeleteSettingNotFound()
         {
-            var connectionString = Environment.GetEnvironmentVariable("AZ_CONFIG_CONNECTION");
-            Assert.NotNull(connectionString, "Set AZ_CONFIG_CONNECTION environment variable to the connection string");
+            string connectionString = SetEnvironment();
```

Comment:
It's a bit strange that a method that returns string is called "Set". Would it be better to call it GetConnectionString?

Reply:
:hushed: yes